### PR TITLE
Initial meta-litmusrt Yocto layer

### PR DESCRIPTION
**Pull request by aanisov**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+=======================
+
+The meta-litmusrt provides minimal LITMUS-RT image:
+	* litmusrt-image-minimal
+
+This is the minimal initramfs image together with LITMUS-RT specific kernel
+together with functional liblitmus and feather-trace-tools.
+This image was intentinally created to provide synthetic RT load for virtualized
+environment (XEN) and measure its reliability, so has correspondent specifics.
+E.g. FT functionality required Python is not available on the system itself, but
```
Comment (andr2000) [inline README]: what does FT stand for?

File: recipes-kernel/linux/linux-litmusrt_4.9.bb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Litmus-RT Linux kernel "
+
+require recipes-kernel/linux/linux-yocto.inc
+
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}/:"
+COMPATIBLE_MACHINE = "aarch64"
+
+RENESAS_BSP_URL = "git://github.com/LITMUS-RT/litmus-rt.git"
```
Comment (andr2000) [inline recipes-kernel/linux/linux-litmusrt_4.9.bb]: aren't we building for generic armv8? why renesas here?

**Comment by aanisov** [comment]
Addressed comments, added a build instruction into README.

**Review by andr2000 — APPROVED** [review]

**Comment by aanisov** [comment]
Guys, I appreciate you find a bit of time to look through this and let it get merged.

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by aanisov** [comment]
@iartemenko be so kind to merge this.

**Pull request merged**